src/services/importService.ts

```diff
@@ -61,33 +65,61 @@ export default class ImportService implements IImportService {
             },
             autoSave: true,
         };
+
+        if (originalProject.visitedFrames.every((item) => typeof item === "number")) {
+            // then this is a video project
+            convertedProject.lastVisitedAssetId = "dummyId";
+        }
+
         return convertedProject;
     }
 
     /**
      * Generate assets based on V1 Project frames and regions
      * @param project - V1 Project Content and File Information
      */
-    public async generateAssets(v1Project: IFileInfo, v2Project: IProject): Promise<IAssetMetadata[]> {
+    public async generateAssets(v1Project: IFileInfo, v2Project: IProject, parent?: IAsset): Promise<IAssetMetadata[]> {
```

Comment:
I suggest we don't pull in the top level video asset here.  The code path for image project or video project should be treated the same in my opinion and the inspection of the project file should happen within this function implementation.